packages/twenty-front/src/modules/object-record/record-filter/utils/isMatchingStringFilter.ts

```diff
@@ -63,6 +63,9 @@ export const isMatchingStringFilter = ({
     case stringFilter.startsWith !== undefined: {
       return value.startsWith(stringFilter.startsWith);
     }
+    case stringFilter.containsAny !== undefined: {
+      return stringFilter.containsAny.some((item) => value.includes(item));
```

Comment:
Getting an error when triple clicking on one of the 9Lol celle
<img width="1504" alt="Screenshot 2024-11-11 at 17 16 13" src="https://github.com/user-attachments/assets/aa28b312-0c15-4a92-9afe-47bd9ba3ca72">

Reply:
I'm not able to reproduce this. I could add a nullish check like so

`return stringFilter.containsAny.some((item) => value?.includes(item));`

but would be better to know why null is passed to this function in the first place. Quick screenshare call?